### PR TITLE
Branch 1.5(changes you requested are done)

### PR DESCRIPTION
**Pull request by blank-27**

### CODE REVIEW

File: Data/participants.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "participants": [
+
```
Comment (dojutsu-user) [inline Data/participants.json]:
```suggestion
```

File: Data/participants.json [hunk 2]
```diff
@@ -27,6 +28,15 @@
      "profilePicture":"https://avatars0.githubusercontent.com/u/55954955?s=460&v=4",
      "facebookLink":"https://www.facebook.com/profile.php?id=100025977280583",
      "githubLink":"https://github.com/yadavashu"
-    }
+
```
Comment (dojutsu-user) [inline Data/participants.json]:
```suggestion
```

File: Data/participants.json [hunk 3]
```diff
@@ -27,6 +28,15 @@
...
+    },
+     {
+
```
Comment (dojutsu-user) [inline Data/participants.json]:
```suggestion
```

File: Data/participants.json [hunk 4]
```diff
@@ -27,6 +28,15 @@
...
+      "name": "Harshdeep",
+      "email": "mottanharshdeep@gmail.com",
+      "profilePicture": "https://avatars0.githubusercontent.com/u/55805028?s=460&v=4",
+      "facebookLink": "https://www.facebook.com/profile.php?id=100010115341689",
+      "githubLink": "https://github.com/blank-27"
+    },
```
Comment (dojutsu-user) [inline Data/participants.json:44]:
```suggestion
    }
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
Please resolve the merge conflicts.

**Comment by blank-27** [comment]
Sorry Sir ,I don't know how to resolve merge conflict.

File: Data/participants.json [hunk 5]
```diff
@@ -34,6 +34,13 @@
      "profilePicture":"https://avatars0.githubusercontent.com/u/55954955?s=460&v=4",
      "facebookLink":"https://www.facebook.com/profile.php?id=100025977280583",
      "githubLink":"https://github.com/yadavashu"
+    },
+     {
+      "name": "Harshdeep",
+      "email": "mottanharshdeep@gmail.com",
+      "profilePicture": "https://avatars0.githubusercontent.com/u/55805028?s=460&v=4",
+      "facebookLink": "https://www.facebook.com/profile.php?id=100010115341689",
+      "githubLink": "https://github.com/blank-27",
```
Comment (dojutsu-user) [inline Data/participants.json:43]:
```suggestion
      "githubLink": "https://github.com/blank-27"
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
@blank-27 
> Sorry Sir ,I don't know how to resolve merge conflict.

That's the whole point of [Get Set FOSS](https://getsetfoss.tech/). You will learn many things that are required for GSoC. You are doing great :tada:. Search your doubts on Google, it will help you a lot.

**Comment by rishabhshuklax** [comment]
You already have #34  opened. First close that then open any new PR for the same issue.

**Pull request closed without merging**